src/libraries/System.Private.Xml/src/System/Xml/Serialization/IXmlSerializable.cs

```diff
@@ -12,6 +13,7 @@ namespace System.Xml.Serialization
     /// </devdoc>
     public interface IXmlSerializable
     {
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
This one seems surprising. Are we sure `GetSchema()` should be EBS.Never?

cc @dotnet/area-system-xml

Reply:
ping @dotnet/area-system-xml

Reply:
cc @StephenMolloy @HongGit

Reply:
This is the last open discussion. Would love to reach conclusion today.

Reply:
I think this falls into the

> For the changes that are just updating src to match the ref, I'd be fine with opening a follow-up issue for each.

bucket. So I guess I wouldn't let this one discussion block you.